src/App.js

```diff
@@ -21,32 +23,66 @@ export default function App() {
     month: '2-digit',
     day: '2-digit',
   });
+  const differenceDate = inputs[0].date.getTime() - inputs[inputs.length - 1].date.getTime();
+  const differenceDays = Math.round(differenceDate / (24 * 3600 * 1000)) + 1;
+  const totalBudget = 30000;
+  const dailyTotalBudget = 1000;
+  const saldoIncrease = inputs[0].increase - dailyTotalBudget;
 
```

Comment:
I noticed that your `App.js` is getting bigger and bigger and is not well readable. I would highly recommend you to refactor your this file.

Reply:
Hey Lene, thanks a lot for your good review: The refactoring fits well with my next userstory. Hope you can give me a hint how I can best proceed the refactoring. Greetings